test/LAMMPS_jll/.pkg/select_artifacts.jl

```diff
@@ -0,0 +1,27 @@
+# Note: This file is executed from `Pkg` in an isolated environment.
+#       We add the current package and stdlib to the load path so that
+#       `platform_augmentation.jl` can use the dependencies of the project,
+#       and `select_artifacts.jl` needs TOML, and Artifacts.
+# Warn: Since this is executed in a sandbox, the current preferences are not
```

Comment:
@staticfloat Another wart I found, but one I am not sure how to best fix.

Reply:
Yeah, using an isolated environment was an intentional design decision, because since these hooks need to be run at `Pkg.add()` time, it's difficult to know what dependencies have already had their artifacts downloaded, etc....  What dependencies beyond stdlibs do you need to load?

Reply:
As an example if we centralize the platform tags for a series of packages in a small package, like my MPIABI, idea then I would need that.